### ISSUE TITLE
Mac 10.15 Ansible PR Builder fails due to incorrect usage of brew cask install

### ISSUE THREAD

**Issue by karianna** [AdoptOpenJDK/openjdk-infrastructure#1790]
```
TASK [Common : Install Build Tool Casks] ***************************************
failed: [localhost] (item=adoptopenjdk10) => {"ansible_loop_var": "item", "changed": false, "item": "adoptopenjdk10", "msg": "Updating Homebrew...\nError: Calling brew cask install is disabled! Use brew install [--cask] instead."}
failed: [localhost] (item=packages) => {"ansible_loop_var": "item", "changed": false, "item": "packages", "msg": "Error: Calling brew cask install is disabled! Use brew install [--cask] instead."}
```

Please set the title to indicate the test name and machine name where known.

To make it easy for the infrastructure team to repeat and diagnose, please
answer the following questions:

- test suite/name?
- Is there an existing issue elsewhere covering this?
- Which machine(s) does it work on?
- Which machine(s) does it fail on?
- Do you have a link to a Grinder re-run if the test with the failure?

Any other details:

**Comment by gdams** [comment]
This is a bug with the upstream ansible cask module. Checking if it's been fixed yet

**Comment by Willsparker** [comment]
It has been fixed - as of Ansible `2.10.5` (see: https://github.com/ansible-collections/community.general/issues/1524#issuecomment-755126741 ). For some reason, however, the PR-Builder installs an older version of Ansible, despite `2.10.5` being the latest version (https://formulae.brew.sh/formula/ansible)

**Comment by Willsparker** [comment]
Okay, as of https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/1870/checks?check_run_id=1782760737 , it appears to be installing the correct version of Ansible. 
New Issue:
```
Run brew untap homebrew/cask-versions
  brew untap homebrew/cask-versions
  brew cask uninstall adoptopenjdk8
  brew cask uninstall adoptopenjdk11
  brew cask uninstall adoptopenjdk12
  brew cask uninstall adoptopenjdk13
  brew cask uninstall adoptopenjdk14
  shell: /bin/bash -e {0}
Error: No available tap homebrew/cask-versions.
Error: Process completed with exit code 1.
```

**Comment by Willsparker** [comment]
Fixed as of #1873
